platform/marvell-arm64/linux-kernel-arm64.mk

```diff
@@ -0,0 +1,8 @@
+# linux kernel package for marvell arm64
+
+KVERSION= 4.4.8
```

Comment:
>4.4.8 [](start = 10, length = 5)

amd64 is already using 4.9.0. Could you use the same or newer version?

Reply:
For Marvell ARM platform we are in process of analysing kernel migration to latest version. 
Since we have tested with 4.4.8, I have retained it

Reply:
I see your concern. Still I prefer the same version with all other platforms. If we introduce different ABI for master branch, some other component may have issues on this platform.

---
In reply to: [292588057](https://github.com/Azure/sonic-buildimage/pull/2980#discussion_r292588057) [](ancestors = 292588057)

Reply:
some sonic features depends on the kernel especially vrf feature, what's your plan to bring kernel to 4.9.0?

Reply:
We have image running with kernel 4.14.76 and is ready now.
Do you still insist on 4.9 version, I have not started working on migrating to 4.9 kernel version.

Reply:
yes, what I am expecting is that we have your arm kernel compile from https://github.com/Azure/sonic-linux-kernel

Reply:
Updated kernel version 4.9.168 as debian package.
In parallel working on to compile kernel from sonic-linux-kernel too.